### DISCUSSION TITLE
Q: deta如何禁用图片上传，deta数据库免费额度有几个G || Q: How to disable image uploading in deta? How many gigabytes is the free quota of deta database?

### DISCUSSION THREAD

**Discussion by ningguoxu** [walinejs/waline#2632] [Q&A] [answered]
### Describe the bug | 问题描述

我觉得图片会占用数据库过多的空间，导致真正有用的评论没有空间存，顺便问一下deta的数据库有多少个G。

### Website URL | 问题网站

https://waline-1-c3938780.deta.app

### Where your waline deploy? | 服务部署在哪里？

Deta

### Where your comment data store? | 数据存储在哪里？

Deta Base

---
### Describe the bug | Problem description

I think pictures will take up too much space in the database, resulting in no space to store really useful comments. By the way, how many gigabytes does the deta database have?

### Website URL | Problem website

https://waline-1-c3938780.deta.app

### Where your waline deploy? | Where is the service deployed?

Deta

### Where your comment data store? | Where is the data stored?

Deta Base

**Comment by track23** [comment, marked as answer]
```js
const waline = init({
  imageUploader: false,
})
```
